### PR TITLE
Nuxt on the port :9000

### PR DESCRIPTION
**Pull request by vikiival**
- :top: nuxt will be running on the port :9000
- set node engine to 14.17

Thank you for your contribution to the KodaDot NFT gallery, 
we really appreciate your contribution!

### PR type
- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### Before submitting this PR, please make sure:
- [x] Your code builds clean without any erros or warnigns
- [ ] You've posted screenshot of demonstrated change in this PR
- [x] Merged recent default branch, **main** and you have no conflicts
- [x] Didn't break any original functionality 

### Optional
- [ ] You've tested it on mobile
- [ ] Are there any edge cases? Name if any 

### What's new? (may be part of changelog)
- closes #<issue_number> - <Brief_description_how_you've_fixed_it>

### If issue had bounty label ($,$$,$$$)
- Fill up your KSM address: https://kodadot.xyz/transfer/?target=<Kusama_Address>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "generate": "nuxt generate"
   },
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=14.17.0"
```
Comment (roiLeo) [inline package.json:14]: why do you need to downgrade node version?

**Review by roiLeo — CHANGES_REQUESTED** [review]
could you please edit DockerFile with new port

File: package.json (hunk 1 above)
Comment (vikiival) [reply package.json:14]: I'm running on 14.17 :D

Comment (roiLeo) [reply package.json:14]: Could you change node version un DockerFile too?

**Review by roiLeo — APPROVED** [review]

**Pull request merged**